src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -70,74 +70,70 @@
  * Example:
  * </p>
  * <pre>
- * public void foo() throws Exception {
- *   int i = 0;
- *   while (i &gt;= 0) {
- *     switch (i) {
- *       case 1:
- *         i++;
- *       case 2: // violation, previous case contains code but lacks
- *               // break, return, yield, throw or continue statement
- *         i++;
- *         break;
- *       case 3: // OK
- *         i++;
- *         return;
- *       case 4: // OK
- *         i++;
- *         throw new Exception();
- *       case 5: // OK
- *         i++;
- *         continue;
- *       case 6: // OK
- *       case 7: // Previous case: OK, case does not contain code
- *               // This case: OK, by default the last case might not have statement
- *               // that transfer control
- *         i++;
+ * class FallThroughTestClass {
+ *   public static void main(String args[]) {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++;
+ *         case 2: // violation, previous case contains code but lacks
+ *           // break, return, yield, throw or continue statement
+ *           i++;
+ *           break;
+ *         case 3: // OK
+ *           i++;
+ *           return;
+ *         case 4: // OK
+ *           i++;
+ *           throw new Exception();
+ *         case 5: // OK
+ *           i++;
+ *           continue;
+ *         case 6: // OK
+ *         case 7: // fall through
+ *         case 8: // Previous case: OK, case does not contain code
+ *           // This case: OK, by default the last case might not have statement
+ *           // that transfer control
+ *           i++;
+ *       }
  *     }
  *   }
  * }
- * public int bar() {
- *   int i = 0;
- *   return switch (i) {
- *     case 1:
- *       i++;
- *     case 2: // violation, previous case contains code but lacks
- *             // break, return, yield, throw or continue statement
- *     case 3: // OK, case does not contain code
- *       i++;
- *       yield 11;
- *     default: // OK
- *       yield -1;
- *   };
- * }
  * </pre>
  * <p>
  * Example how to suppress violations by comment:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     i++; // fall through
- *
- *   case 2: // OK
- *     i++;
- *     // fallthru
- *   case 3: { // OK
- *     i++;
+ * class FallThroughTestClass {
+ *   public static void main(String args[]) {
```

Comment:
Please remove whole block `Example how to suppress violations by comment:` it is not required. We show suppression in Example1.

Reply:
done.